device/globals/game.gd

```diff
@@ -373,28 +373,16 @@ func set_camera_limits():
 
 func load_hud():
 	var hres = vm.res_cache.get_resource(vm.get_hud_scene())
-	get_node("hud_layer/hud").replace_by_instance(hres)
-
-	# Add inventory to hud layer, usually hud_minimal.tscn, if found in project settings
-	if !$hud_layer.has_node("inventory") and ProjectSettings.get_setting("escoria/ui/inventory"):
-		inventory = load(ProjectSettings.get_setting("escoria/ui/inventory")).instance()
-		if inventory and inventory is preload("res://globals/inventory.gd"):
-			inventory.hide()
-			$hud_layer.add_child(inventory)
-	else:
-		inventory = get_node("hud_layer/hud/inventory")
-
-	# Add action menu to hud layer if found in project settings
-	if ProjectSettings.get_setting("escoria/ui/action_menu"):
-		action_menu = load(ProjectSettings.get_setting("escoria/ui/action_menu")).instance()
-		if action_menu and action_menu is preload("res://globals/action_menu.gd"):
-			$hud_layer.add_child(action_menu)
+	if get_node("hud_layer/hud") is InstancePlaceholder:
+		get_node("hud_layer/hud").replace_by_instance(hres)
```

Comment:
The hud has already been instanced in the `game_am.tscn`. I will check to see why that might be.